go-controller/pkg/cni/helper_linux.go

```diff
@@ -315,6 +322,19 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 		}
 	}
 
+	err = netns.Do(func(hostNS ns.NetNS) error {
+		if _, err := os.Stat("/proc/sys/net/ipv6/conf/all/dad_transmits"); !os.IsNotExist(err) {
+			err = setSysctl("/proc/sys/net/ipv6/conf/all/dad_transmits", 0)
+			if err != nil {
+				return fmt.Errorf("failed to disable IPv6 DAD: %q", err)
```

Comment:
Should this be an error, or just a warning? If it fails, we can continue...

Reply:
good point.  a warning makes sense.